apps/ledger-live-desktop/src/renderer/families/celo/VoteFlowModal/fields/ValidatorGroupsField.js

```diff
@@ -0,0 +1,115 @@
+// @flow
+import { getAccountUnit } from "@ledgerhq/live-common/lib/account";
+import type { CeloValidatorGroup } from "@ledgerhq/live-common/lib/families/celo/types";
+import type { Account, TransactionStatus } from "@ledgerhq/live-common/lib/types";
+import invariant from "invariant";
+import React, { useEffect, useMemo, useRef, useState } from "react";
+import { Trans } from "react-i18next";
+import styled from "styled-components";
+import Box from "~/renderer/components/Box";
+import ScrollLoadingList from "~/renderer/components/ScrollLoadingList";
+import Text from "~/renderer/components/Text";
+import IconAngleDown from "~/renderer/icons/AngleDown";
+import type { ThemedComponent } from "~/renderer/styles/StyleProvider";
+import ValidatorGroupRow from "../components/ValidatorGroupRow";
+
+import { useCeloPreloadData } from "@ledgerhq/live-common/lib/families/celo/react";
+
+type Props = {
+  account: Account,
+  status: TransactionStatus,
+  chosenValidatorGroupAddress: ?string,
+  onChangeValidatorGroup: (v: CeloValidatorGroup) => void,
+};
+
+const ValidatorGroupsField = ({
+  account,
+  onChangeValidatorGroup,
+  chosenValidatorGroupAddress,
+  status,
+}: Props) => {
+  invariant(account && account.celoResources, "celo account and resources required");
+
+  const [showAll, setShowAll] = useState(false);
+
+  const unit = getAccountUnit(account);
+
+  const { validatorGroups } = useCeloPreloadData();
+
+  const chosenValidatorGroup = useMemo(() => {
+    if (chosenValidatorGroupAddress !== null) {
+      return validatorGroups.find(v => v.address === chosenValidatorGroupAddress);
+    }
+  }, [validatorGroups, chosenValidatorGroupAddress]);
+
+  const containerRef = useRef();
+
+  /** auto focus first input on mount */
+  useEffect(() => {
+    /** $FlowFixMe */
```

Comment:
I could be wrong, but I don't think there's a type error on the line below.  If that's true, we can remove the $FlowFixMe.

Reply:
yeah, another copy paste, thanks